.github/workflows/build_cmake.yml

```diff
@@ -22,6 +22,13 @@ jobs:
       fail-fast: false
       matrix:
         config:
+        - {
+            name: "Ubuntu Latest Clang", artifact: "",
+            os: ubuntu-latest,
+            cc: "clang", cxx: "clang++",
+            cmake_flags: "-G Ninja -DENABLE_TESTING=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=1",
```

Comment:
CMake 3.6 or later supports [`CMAKE_CXX_CLANG_TIDY`](https://cmake.org/cmake/help/v3.6/variable/CMAKE_LANG_CLANG_TIDY.html) variable.
With `-DCMAKE_CXX_CLANG_TIDY=clang-tidy;--fix`, clang-tidy fixes source files if possible.

Reply:
Today Iearned .. 😄

Reply:
Note that multiple build processes launched by ninja may modify a header file at the same time, which messes the file and results in build failure.

Reply:
So, the serial script would be preferable to avoid a spurious build failure.